### ISSUE TITLE
Error due to update in meteor-node-stubs

### ISSUE THREAD

**Issue by MKRazz** [meteor-vue/vue-meteor#307]
meteor-node-stubs was just updated replacing `http-browserify` with `stream-http` (https://github.com/meteor/node-stubs/commit/488059954d67e397432dea3b6dbe7696ee82939e)

This update is causing a client error in `vue-component-dev-client` due to this:

https://github.com/meteor-vue/vue-meteor/blob/41228c434e96766f8ff2bc4b9e6b02a8546116a8/packages/vue-component-dev-client/client/dev-client.js#L8-L9

**Comment by DirkStevens** [comment]
I have the same problem...

@MKRazz did you find a work-around?

**Comment by MKRazz** [comment]
@DirkStevens Yep, `meteor-node-stubs` gets added to your project's dependencies, so change the version from `^0.3.2` to `0.3.2` then `meteor npm i`.

**Comment by DirkStevens** [comment]
@MKRazz Thanks so much. that works!

**Comment by mitar** [comment]
See: https://github.com/meteor-vue/vue-meteor/pull/311

**Comment by mitar** [comment]
With what was this fixed? I cannot fix a code reference?

**Comment by Akryum** [comment]
The code was removed as it seems it was an issue between socket.io and http-browserify.
